README.md

```diff
@@ -23,9 +23,18 @@ The built images are available from [Docker Hub][hub]
 - `docker.io/appuio/oc:v3.10`
 - `docker.io/appuio/oc:v3.11`
 
+## Caveats
+
+* Newer versions of this image do not include `image-cleanup` anymore.
```

Comment:
The task deliverables in VINT-1816 say, we would deprecate the `image-cleanup` command first. You could add a script that wraps the binary, issuing the deprecation warning (so we wouldn't need to update the binary).

Reply:
That would mean we keep an old installation of image cleanup installed and also install seiso, but I guess there's no way around it.

Reply:
That's exactly what "deprecation" mean, yes. We keep `image-cleanup` around.

Not my choice. We agreed on that in the backlog refinement.

Reply:
I'm fine with that, I have no hard feelings